src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -1204,6 +1204,98 @@
         az acr webhook update -n MyWebhook -r MyRegistry --status disabled
 """
 
+# region connected-acr TODO
+helps['acr connected-acr'] = """
+type: group
+short-summary: Manage Edge/On-Prem Container Registries with Azure Container Registries.
+"""
+
+helps['acr connected-acr create'] = """
+type: command
+short-summary: Create a connected registry for an Azure Container Registry.
+examples:
+  - name: Create a connected registry with DNS name 'edge01.contoso.internal' that has 'mycloudregistry.azurecr.io' as a parent.
+    text: |
+        az acr connecter-acr create --registry mycloudregistry.azurecr.io 
+            --name Contoso-Edge01-in-Wichita --repositories "contoso-app/* contoso-service/*"
+  - name: Create a mirror connected registry with only read permissions.
+    text: |
+        az acr connecter-acr create --registry mycloudregistry.azurecr.io
+            --mode mirror --parent Contoso-Edge01-in-Wichita
+            --name Contoso-Mirror01-in-Wichita --repositories contoso-app/hello-world
+  - name: Create a mirror connected registry with auto update disable, that syncs every day at midninght and sync window of 4 hours.
```

Comment:
For `--sync-window` and `--sync-message-ttl` the command descriptions ask for the values in ISO8601 duration format. Please update the examples to use this format. For instance, `4H` -> `PT4H`, `90D` -> `P90D`.